.github/workflows/codeql-analysis.yml

```diff
@@ -39,6 +39,8 @@ jobs:
     steps:
     - name: Checkout repository
       uses: actions/checkout@v3
+      with:
+        submodules: recursive
```

Comment:
Is this still required given we don't have a git submodule anymore?

Reply:
Yeap, not needed.